vite.md

```diff
@@ -452,3 +456,128 @@ node bootstrap/ssr/ssr.mjs
 ```
 
 > {tip} Laravel's [starter kits](/docs/{{version}}/starter-kits) already include the proper Laravel, Inertia SSR, and Vite configuration. Check out [Laravel Breeze](/docs/{{version}}/starter-kits#breeze-and-inertia) for the fastest way to get started with Laravel, Inertia SSR, and Vite.
+
+<a name="script-and-style-attributes"></a>
+## Script & Style Tag Attributes
+
+<a name="content-security-policy-csp-nonce"></a>
+### Content Security Policy (CSP) Nonce
+
+If you wish to include a [`nonce` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce) on your script and style tags as part of your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP), you may generate or specify a nonce using the `useCspNonce` method within a custom [middleware](/docs/{{version}}/middleware):
+
+```php
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Vite;
+
+class AddContentSecurityPolicyHeaders
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        Vite::useCspNonce();
+
+        return $next($response)->withHeaders([
```

Comment:
@jessarcher  Shouldn't this be `return $next($request)->withHeaders([` ?

Reply:
Good catch, thank you!

PR to fix it: https://github.com/laravel/docs/pull/8097